test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperFormExtensionsTest.cs

```diff
@@ -397,10 +397,14 @@ public void BeginFormWithMethodParameter_CallsHtmlGeneratorWithExpectedValues(Fo
                     null))  // htmlAttributes
                 .Returns(tagBuilder)
                 .Verifiable();
-            htmlGenerator
-                .Setup(g => g.GenerateAntiforgery(htmlHelper.ViewContext))
-                .Returns(HtmlString.Empty)
-                .Verifiable();
+
+            if (method != FormMethod.Get)
+            {
+                htmlGenerator
+                    .Setup(g => g.GenerateAntiforgery(htmlHelper.ViewContext))
+                    .Returns(HtmlString.Empty)
+                    .Verifiable();
+            }
```

Comment:
Discussed offline. Since you're not creating new tests and it wouldn't be a quick change, fine as-is.